src/app/lib/utilities/rendererEnvHelper/index.js

```diff
@@ -0,0 +1,10 @@
+import { getParsedQueryString } from '../urlParser';
+import isLive from '../envHelper';
+
+const hasLiveOverride = url => {
+  const params = getParsedQueryString(url);
+
+  return params && !isLive() && params.renderer_env === 'live';
```

Comment:
I had a look at getParsedQueryString and saw this `should return empty query string object from url when no query string exists` so that means `params` will always be true

Reply:
maybe something like 

```js
export default url => {
  if (isLive()) {
    return false;
  }
  const { renderer_env } = getParsedQueryString(url);

  return renderer_env === 'live';
};
```

Reply:
Shall I just remove the check for `params`, or check `params != ''`

Reply:
might be my jet lagged brain atm but `getParsedQueryString` sounds a little odd to me. what about `parseQueryString` or `getQueryParams`?

Reply:
I have gone for getQueryParams - there is already getQueryString function which returns a string, so this makes sense

Reply:
still the issue that params will always be truthy

Reply:
> Shall I just remove the check for `params`, or check `params != ''`

does getParsedQueryString every return a string?

Reply:
Yes - if there are no query params, getQueryParams returns `''`, otherwise, it returns an object containing the params as name value pairs

Reply:
this test isn't accurate then `should return empty query string object from url when no query string exists` and it's not a good idea for a function to have a dynamic return type

Reply:
Oops, I was wrong - it returns an empty object `{}` rather than an empty string. 
https://github.com/bbc/simorgh/blob/3b3615c4308369745d58068a3634402789831f19/src/app/lib/utilities/urlParser/index.test.js#L46

Reply:
I know you merged it but can we still fix this code then?

Reply:
Already done -> https://github.com/bbc/simorgh/blob/latest/src/app/lib/utilities/rendererEnvHelper/index.js - is this what you meant?